docs/api-guide.md

```diff
@@ -697,17 +689,28 @@ Put put =
         .build();
 ```
 
-#### `Delete` operation
+##### Disable implicit pre-read for `Put` operations
 
-`Delete` is an operation to delete a record specified by a primary key.
+If you're certain that a record you're trying to write doesn't exist, you can disable implicit pre-read for the `Put` operation for better performance. A `Put` without implicit pre-read is faster than a regular `Put` as it skips the implicit pre-read. However, if the record does exist, the operation will fail due to a conflict.
```

Comment:
> If you're certain that a record you're trying to write doesn't exist, you can disable implicit pre-read for the `Put` operation for better performance.

This describes the implicit pre-read feature can be disabled. But there is no explanation in advance about what "implicit pre-read" is. How about adding a short description about "implicit pre-read" mechanism? It may be helpful for those who are interest in ScalarDB performance, but don't know implicit pre-read happens by default.

Reply:
Thank you for pointing that out. That's right. I've added a short description about "implicit pre-read" in a7c007146b5501822088b4e950449b551d357dd2. Please take a look! Thanks.